File: src/정렬/boj_2822.py
l = sorted([(int(input()), i) for i in range(1, 9)], key=lambda t: t[0])[3:]
print(sum(map(lambda t: t[0], l)), " ".join(map(str, sorted(map(lambda t: t[1], l)))), sep="\n")

File: src/정렬/boj_2822.js
const fs = require("fs");
const stdin = fs.readFileSync("/dev/stdin").toString().trim().split("\n");
const input = (() => {
    let line = 0;
    return () => stdin[line++];
})();
const arr = [];
for (let i = 1; i <= 8; i++) {
    arr.push([+input(), i]);
}

const target = arr.sort((a, b) => a[0] - b[0]).slice(3);
console.log(target.reduce((acc, cur) => acc + cur[0], 0));
console.log(
    target
        .map((v) => v[1])
        .sort((a, b) => a - b)
        .join(" "),
);
